docs/agave/guides/systems/introduction.rst

```diff
@@ -134,7 +137,7 @@ List only private systems
 
 .. code-block:: plaintext
 
-   systems-list -v -Q
+   tapis systems search --public eq FALSE
```

Comment:
Says it's supposed to list private systems but the search states public

Reply:
yes, with public set as false, it will print the private sytems